static/src/javascripts/projects/common/modules/social/share-count.js

```diff
@@ -86,12 +86,12 @@ define([
             var url = 'http://www.theguardian.com/' + config.page.pageId;
             try {
                 ajax({
-                    url: 'https://graph.facebook.com/' + url,
+                    url: 'https://graph.facebook.com/' + url, //TODO: use recent Graph API endpoint format (versioned) https://developers.facebook.com/docs/graph-api/reference/v2.7/url
                     type: 'json',
                     method: 'get',
                     crossOrigin: true
                 }).then(function (resp) {
-                    var count = resp.shares || 0;
+                    var count = resp.share.share_count || 0;
```

Comment:
This is slightly more likely to throw if facebook change their API again. If we're happy for this to go to the `errorReporter` then it's fine, but if we really want it to default to 0 then `var count = resp.share && resp.share.share_count || 0` is an option.